docs-client/src/lib/json-util.ts

```diff
@@ -134,3 +136,17 @@ export function isValidJsonMimeType(applicationType: string | null) {
   }
   return applicationType.indexOf('json') >= 0;
 }
+
+export function extractHeaderLines(headers: Headers): string[] {
+  const headerLines: string[] = [];
+  headers.forEach((value, name) => {
+    if (listHeaders.has(name.toLowerCase())) {
+      value.split(',').forEach((v) => {
+        headerLines.push(`${name}: ${v.trim()}`);
+      });
+    } else {
+      headerLines.push(`${name}: ${value}`);
+    }
+  });
```

Comment:
Question) Is there even a need for this logic? How headers set multiple values can be implementation specific, and I don't see the need for us to maintain a separate list of headers.

```suggestion
  headers.forEach((value, name) => {
    headerLines.push(`${name}: ${value}`);
  });
```

Reply:
I refered to RFC 9110 and serveral implementations, but found that the handling of multi-value headers are slightly different across them. so, maintaining a manual list of exceptions may not be necessary